### ISSUE TITLE
Xcode 11 Crash on Home Screen

### ISSUE THREAD

**Issue by xthemadgenius** [John-Lluch/SWRevealViewController#816]
I have all my code set up for SWRevealViewController it worked perfectly fine on Xcode 10.3 and even on Xcode 11.2 beta 2 but for some reason its not working on the recent update release from the App Store and I end up getting this error 

Thread 1: EXC_BAD_ACCESS (code=1, address=0x48)

at the app delegate screen after I toggle with my SWRevealView Controller. any ideas how I could get this fixed

**Comment by iDevelopper** [comment]
Do you have the exception details?

**Comment by xthemadgenius** [comment]
what do you mean by exception details?

**Comment by djerick1989** [comment]
Hi XaviXO, did you solve this issue?, i have a similar problem with IOS 13.1 XCode 11.1, but i don´t receive any error message only the screen still in black.

```
let storyboard = UIStoryboard(name: "Main", bundle: nil)
        var vc = storyboard.instantiateViewController(withIdentifier: "inicio") as! SWRevealViewController
            
        self.window?.rootViewController = vc
        self.window?.makeKeyAndVisible()
```

**Comment by iDevelopper** [comment]
Have you setted a front view controller?

**Comment by djerick1989** [comment]
Yes, my application works in all versions except IOS 13, but only when i select the target IOS 12.4

 I document my case this forum. https://forums.developer.apple.com/message/390323#390323

**Comment by ralspatel** [comment]
Hi @djerick1989 

I have the same issue that you are facing
When login successfully I will navigate to SWRevealViewController, same way i change rootViewcontroller from AppDelegate file, but it always open black screen.

please help me to solve this issue.

Thanks

**Comment by djerick1989** [comment]
Hi @ralspatel 

Review my answer in stackoverflow.

https://stackoverflow.com/questions/58207755/ios13-on-xcode-11-black-screen-even-after-adding-scenedelegate-and-updating-info/58565994#58565994
